src/braket/default_simulator/simulator.py

```diff
@@ -383,7 +388,9 @@ def _observable_hash(observable: Observable) -> Union[str, dict[int, str]]:
             return str(observable.__class__.__name__)
 
     @staticmethod
-    def _formatted_measurements(simulation: Simulation) -> list[list[str]]:
+    def _formatted_measurements(
+        simulation: Simulation, measured_qubits: list[int] = None
```

Comment:
```suggestion
        simulation: Simulation, measured_qubits: list[int] | None = None
```
and in the doc string